src/Lykke.Logs/Obsolete/LogLevel.cs

```diff
@@ -5,6 +5,7 @@ namespace Lykke.Logs
     /// <summary>
     /// Specifies log level
     /// </summary>
+    [Obsolete("Use new Lykke logging system")]
```

Comment:
Can we replace this to cref of a type from new logging system?

Reply:
fixed